### DISCUSSION TITLE
Too many updates

### DISCUSSION THREAD

**Discussion by wcrabtree9** [mhx/dwarfs#197] [General]
Please, cool it with the updates! I'm on Arch and this damn thing has been popping up every other day and takes forever to update.

**Comment by xcfmc** [comment]
I'm enjoying the updates. Maybe there's a user side fix for your issue.

**Comment by silentnoodlemaster** [comment]
frequent updates are to be expected while in version 0.x.x, as for arch linux, there is a prebuild binary package available too

**Comment by mhx** [comment]
Don't worry, the frequent updates over the last couple of days were quite unusual. Unless there are any significant bugs, there shouldn't be too many updates in the near future.

**Comment by bionade24** [comment]
> and takes forever to update.

You can use my repo & CD if you trust me. https://wiki.archlinux.org/title/Unofficial_user_repositories#oscloud
Package build status: https://abs-cd.oscloud.info/cd_manager/dwarfs
CI/CD software: https://github.com/bionade24/abs_cd
